deep_autoviml/deep_autoviml.py

```diff
@@ -130,7 +131,9 @@ def left_subtract(l1,l2):
 ##############################################################################################
 def fit(train_data_or_file, target, keras_model_type="basic", project_name="deep_autoviml",
                                 save_model_flag=True, model_options={}, keras_options={},
-                                 use_my_model='', model_use_case='', verbose=0):
+                                 use_my_model='', model_use_case='', verbose=0,
+                                 use_mlflow=False,mlflow_exp_name='autoviml',mlflow_run_name='first_run'

```

Comment:
This could have been under a single `mlflow_options` dict, similar to `model_options` and `keras_options` args.